receiver/vcenterreceiver/metrics.go

```diff
@@ -57,6 +57,12 @@ func (v *vcenterMetricScraper) recordVMUsages(
 	swappedMem := vm.Summary.QuickStats.SwappedMemory
 	swappedSSDMem := vm.Summary.QuickStats.SsdSwappedMemory
 
+	if memUsage > 0 {
+		totalMemory := vm.Summary.Config.MemorySizeMB
+		memoryUtilization := float64(memUsage) / float64(totalMemory) * 100
```

Comment:
Just to be safe, we should check `if totalMemory == 0`

Reply:
Sure. Added inthe latest. I rebased off main and regenerated the test yaml files so hopefully all good now.